Styling

donacrisante opened the pull request.

donacrisante committed:
import image for circular progress bar

donacrisante committed:
try to put background image inside circular progress bar

donacrisante committed:
add background to circular progress + styling to button on homepage and navbar

donacrisante committed:
style entry form

donacrisante committed:
style entry list and chart

donacrisante committed:
style buttons

donacrisante committed:
adjust place buttons

donacrisante committed:
implement toast message for edit and delete function

donacrisante committed:
add special characters in pattern

donacrisante committed:
uninstall toastify

donacrisante commented:
For the edit and delete function I have replaced the success window with a toast message. See Toast.js and app.js.

Kheirah commented on components/Badge/Badge.js:
```diff
@@ -1,9 +1,18 @@
 export default function Badge({ children, isActive }) {
     return (
-      <span name={`badge${isActive ? " badge--active" : ""}`}>
+      <StyledBadge name={`badge${isActive ? " badge--active" : ""}`}>
```
The `name` attribute doesn't do anything here.

Kheirah commented on components/EntryList/EntryList.js:
```diff
@@ -40,38 +41,87 @@ export default function EntryList({
+  margin-block-start: 0em;
+`;
+
+const StyledIcon = styled.button`
```
It's still a `button` that you style here, not an icon =)

Kheirah commented on components/FavouriteButton/FavouriteButton.js:
```diff
@@ -2,34 +2,27 @@ import StarFilled from "../StarFilled/StarFilled.js";
+}) {
+  return (
+    <FavButton
+      name="favourite-button"
```
Unless there's a reason why the `name` attribute is here, you may safely remove it.

Kheirah commented on components/NavBar/NavBar.js:
```diff
@@ -6,34 +6,46 @@ export default function NavBar() {
+  border-bottom: 0px;
+`;
+
+const Icon = styled.div`
```
It's not an icon just because you say so. It's more like a wrapper for an icon.

Kheirah commented on components/Tab/Tab.js:
```diff
@@ -1,14 +1,25 @@
-  }
-
+  return (
+    <StyledTab name={`tab${isActive ? " tab--active" : ""}`} onClick={onClick}>
```
The same applies here for the `name` attribute. It doesn't do anything.

Kheirah commented on components/Tabs/Tabs.js:
```diff
@@ -1,5 +1,13 @@
 
 export default function Tabs({ children }) {
-  return <div name="tabs">{children}</div>;
+  return <StyledTabs name="tabs">{children}</StyledTabs>;
```
The `name` attribute doesn't do anything here either.

Kheirah reviewed (approved):
There are some `name` attributes from previous user stories, which you may remove. As far as I can tell, you don't need them.

donacrisante commented:
Thank you, Ernst, for your Feedback. I have removed the useless name attributes and corrected the of some styled components (StyledButton instead of StyledIcon and IconContainer instead of Icon).

donacrisante committed:
remove useless name attributes and change name of some styled components
